vm/safepoints.cpp

```diff
@@ -35,11 +35,18 @@ void factor_vm::handle_safepoint(cell pc) {
   faulting_p = false;
 
   if (atomic::load(&safepoint_fep_p)) {
+    atomic::store(&safepoint_fep_p, false);
```

Comment:
@bjourne is there any problem with moving this store to the top of this block?  I would think it would be better to leave it at the end.